### ISSUE TITLE
Error when using ensureDependencies

### ISSUE THREAD

**Issue by akanix42** [maxnowack/meteor-globals#21]
When I use `ensureDependencies` (Meteor 1.5), I receive the following error:
```
W20170927-15:04:16.558(-5)? (STDERR) /home/nathantreid/.meteor/packages/meteor-tool/.1.5.2_1.y8vio++os.linux.x86_64+web.browser+web.cordova/mt-os.linux.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20170927-15:04:16.559(-5)? (STDERR) 						throw(ex);
W20170927-15:04:16.560(-5)? (STDERR) 						^
W20170927-15:04:16.561(-5)? (STDERR) 
W20170927-15:04:16.561(-5)? (STDERR) Error: @go2/meteor-security-helpers has to be installed inside a meteor project!
W20170927-15:04:16.562(-5)? (STDERR)     at Object.ensureDependencies (/home/nathantreid/projects/go2/swmid-228671-mp/site/node_modules/meteor-globals/dist/ensureDependencies.js:41:24)
W20170927-15:04:16.562(-5)? (STDERR)     at Object.<anonymous> (/home/nathantreid/projects/go2/swmid-228671-mp/site/node_modules/@go2/meteor-security-helpers/dist/index.js:4:18)
W20170927-15:04:16.563(-5)? (STDERR)     at Module._compile (module.js:409:26)
W20170927-15:04:16.565(-5)? (STDERR)     at Object.Module._extensions..js (module.js:416:10)
W20170927-15:04:16.566(-5)? (STDERR)     at Module.load (module.js:343:32)
W20170927-15:04:16.566(-5)? (STDERR)     at Function.Module._load (module.js:300:12)
W20170927-15:04:16.567(-5)? (STDERR)     at Module.require (module.js:353:17)
W20170927-15:04:16.569(-5)? (STDERR)     at require (internal/module.js:12:17)
W20170927-15:04:16.570(-5)? (STDERR)     at npmRequire (/home/nathantreid/projects/go2/swmid-228671-mp/site/.meteor/local/build/programs/server/npm-require.js:133:10)
W20170927-15:04:16.570(-5)? (STDERR)     at Module.useNode (packages/modules-runtime.js:687:18)
=> Exited with code: 1
=> Your application is crashing. Waiting for file change.
```

I explored [`checkMeteor.js`](https://github.com/maxnowack/meteor-globals/blob/master/src/checkMeteor.js) a bit and checked the `packageFile` path. The call `path.resolve('./.meteor/packages')` is resolving to the path `/home/nathantreid/projects/go2/swmid-228671-mp/site/.meteor/local/build/programs/server/.meteor/packages`. As you can see, it's resolving to the Meteor build directory rather than the Meteor project directory. For now, I've worked around this behavior by modifying [the return](https://github.com/maxnowack/meteor-globals/blob/master/src/checkMeteor.js#L12) so that if the package file existence check fails, it checks to see if it's resolved under a `.meteor` directory:
``` js
    return fs.existsSync(packageFile) || packageFile.replace(/\\/g, '/').match('/.meteor/');
```

The `.replace` is to ensure that it works on Windows too.

**Comment by maxnowack** [comment]
Thanks for reporting this issue. It seems that it isn't easy as I thought to get the meteor root folder.
It should be fixed when #22 gets merged and released.

**Comment by maxnowack** [comment]
fixed in v1.1.2
